pages/common/xplr.md

```diff
@@ -0,0 +1,24 @@
+# xplr
+
+> Explore the file system on your terminal.
```

Comment:
Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:
```suggestion
> Terminal-based file system explorer.
```